install/kubernetes/helm/istio/templates/configmap.yaml

```diff
@@ -117,6 +118,24 @@ data:
     # this flag is ignored if enableSdsTokenMount is set.
     # This isn't supported for non-k8s case.
     sdsUseK8sSaJwt: {{ .Values.global.sds.useNormalJwt }}
+    {{- else }}
+    # Set expected values when SDS is disabled
+    # Unix Domain Socket through which envoy communicates with NodeAgent SDS to get
+    # key/cert for mTLS. Use secret-mount files instead of SDS if set to empty.
+    sdsUdsPath: ""
```

Comment:
@ostromart  is working on a 'mesh config' proto based on values.yaml, and the goal is to use it for all configs.
I don't remember if it includes the old configmap fields - but sds settings appear to be duplicated in configmap and values.yaml

We are working on improving the UX for SDS installation and to support opt-in/gradual - there are few issues filed on this area, but the current (alpha) settings will likely need to be changed. In particular we will set the sdsUdsPath 
to have a default value in all cases, will have code in pilot-agent to auto-detect if the pod runs on a node with a valid
UDS path and fallback to secrets if not.

And in particular we want to minimize the number of steps and settings for the user - and reduce the confusion due to different words used in different files.  

The 'sds.enabled' flag and types of tokens to allow will only be needed in citadel. 
Pilot will generate SDS config whenever a pod has a SDS UDS socket, and injector will use an opt-in annotation
or global sds.enabled.  

Not clear if we want to allow the sdsUdsPath to be changed ( given the complexity on injector template ), but no 
reason to ever set it to "". 

I would recommend closing this PR and focusing on the UX/upgrade improvements and cleaning up.